### ISSUE TITLE
User specific subscription question

### ISSUE THREAD

**Issue by wmaethner** [fhircast/docs#18]
Given the scenario:
Two instances of an EHR (a desktop application and a mobile application) that a user is logged into.
A subscribing application subscribes to a user specific UserSession changes (when the user changes context inform them). 
The user changes patient context on the desktop, the patient context changes in the subscribing app. Good.
The user then changes patient context on the mobile app, the patient context changes in the subscribing app. Good.
The patient context is different between the desktop and mobile app for the EHR, but now there is a difference in context between the EHRs apps and the subscribing app.

This could be a realistic workflow in an ER setting we believe, the ER doctor is at their desktop with a mobile version of the PACS that should be in sync. They also have a mobile version of the EHR that they also want to stay in sync with the PACS.

Questions:
- Is this on the EHR to keep their two instances in sync?
- Do we need two separate instances of the subscribing app?
- Does the EHR restrict user specific subscriptions to a single instance?
- Is this an unrealistic situation or expectation?

**Comment by IntegratorBrad** [comment]
No, this is not unrealistic. There are many places in the world (Canada, for instance), where there are HIEs or jurisdictional VNAs where it is pivotal that context is shared locally and with the remote app. I'd say the "context synchronization" is far worse than "inter-workstation context synchronization".
